keycloak-test/realms/moh_citizen/clients/emaccs/main.tf

```
@@ -25,4 +25,24 @@ resource "keycloak_openid_client" "CLIENT" {
   ]
   web_origins = [
   ]
-}
\ No newline at end of file
+}
+resource "keycloak_openid_user_attribute_protocol_mapper" "idir_user_guid" {
+  add_to_access_token = true
+  add_to_id_token     = true
+  add_to_userinfo     = true
+  claim_name          = "idir_user_guid"
+  client_id           = keycloak_openid_client.CLIENT.id
+  name                = "idir_user_guid"
+  user_attribute      = "siteminder_user_guid"
+  realm_id            = keycloak_openid_client.CLIENT.realm_id
+}
```

We're not consistent, but I do like one line of white space between resource declarations.

```suggestion
}

```